etcdctl/ctlv3/command/member_command.go

```diff
@@ -111,8 +112,34 @@ func memberAddCommandFunc(cmd *cobra.Command, args []string) {
 	if err != nil {
 		ExitWithError(ExitError, err)
 	}
+	newID := resp.Member.ID
 
 	display.MemberAdd(*resp)
+
+	ctx, cancel = commandCtx(cmd)
+	listResp, err := mustClientFromCmd(cmd).MemberList(ctx)
```

Comment:
Could you refractor this logic out from https://github.com/coreos/etcd/blob/master/etcdctl/ctlv3/command/member_command.go#L167-L172 as a helper function to get list of member?
Maybe name it `getMemberList()`